stacker/providers/aws/default.py

```diff
@@ -362,30 +363,63 @@ def create_change_set(cfn_client, fqn, template, parameters, tags,
     return changes, change_set_id
 
 
+def check_tags_present(actual, expected):
```

Comment:
could you write a small docstring for this for the api docs?

Reply:
Sure, will do.

Reply:
Done.